astropy/io/fits/diff.py

```diff
@@ -352,7 +378,8 @@ def _report(self):
                     wrapper.fill(ignore_fields)))
         self._writeln(u(' Maximum number of different data values to be '
                         'reported: {}').format(self.numdiffs))
-        self._writeln(u(' Data comparison level: {}').format(self.tolerance))
+        self._writeln(u(' Relative tolerance: {},'
```

Comment:
This does not add a newline between these two values, if that's wanted you need to add a `\n` at the end of the string.